src/main/java/com/google/sps/data/Restaurant.java

```diff
@@ -25,13 +25,16 @@
     private final double rating;
     @SerializedName(value = "businessStatus", alternate = "business_status")
     private final String businessStatus;
+    @SerializedName(value = "priceLevel", alternate = "price_level")
+    private final int priceLevel;
     private final Photo[] photos;
 
-    public Restaurant(String name, double rating, String businessStatus, Photo[] photos) {
+    public Restaurant(String name, double rating, String businessStatus, int priceLevel, Photo[] photos) {
```

Comment:
nit: why priceLevel and not price? 
jw since you have getPrice() below

Reply:
changed to price